guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleConnection.java

```diff
@@ -157,6 +162,43 @@ public SimpleConnection(String name, String identifier,
         this.fullConfig = config;
         this.interpretTokens = interpretTokens;
 
+    }
+    
+        /**
+     * Creates a new SimpleConnection having the given identifier,
+     * GuacamoleConfiguration, and GuacamoleProxyConfiguration. Parameter tokens
+     * will be interpreted if explicitly requested.
+     *
+     * @param name
+     *     The name to associate with this connection.
+     *
+     * @param identifier
+     *     The identifier to associate with this connection.
+     *
+     * @param config
+     *     The configuration describing how to connect to this connection.
+     *
+     * @param interpretTokens
+     *     Whether parameter tokens in the underlying GuacamoleConfiguration
+     *     should be automatically applied upon connecting. If false, parameter
+     *     tokens will not be interpreted at all.
+     * 
+     * @param proxyConfig
+     *     The Guacamole proxy configuration used to tell the client how to
+     *     connect to guacd.
+     */
+    public SimpleConnection(String name, String identifier,
+            GuacamoleConfiguration config, boolean interpretTokens,
+            GuacamoleProxyConfiguration proxyConfig) {
+
+        super.setName(name);
+        super.setIdentifier(identifier);
+        super.setConfiguration(config);
+
+        this.fullConfig = config;
+        this.interpretTokens = interpretTokens;
```

Comment:
I think we should be DRY here and just `this(name, identifier, config, interpretTokens);`.